priv/apns/dev_cert.pem

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can all these `*.pem` files be generated on demand? For example when the project is first compiled or run?